index.js

```diff
@@ -34,6 +34,13 @@ Metromatic.instrument = function (object, options) {
         }
       });
     }
+
+    if (metric.type === 'gauge') {
+      metric.events = metric.events || {};
```

Comment:
This default-set line could be moved outside this block and remove the on in the 'timing' block also.